packages/sources/react/src/components/VtmnLoader/VtmnLoader.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import '@vtmn/css-loader/dist/index-with-vars.css';
+import { VtmnLoaderSize } from './types';
+
+export interface VtmnLoaderProps extends React.ComponentPropsWithoutRef<'div'> {
+  /**
+   * The size of the loader.
+   * @default 'medium'
+   */
+  size?: VtmnLoaderSize;
+}
+
+export const VtmnLoader = ({
+  size = 'medium',
```

Comment:
Is it possible to use an enumeration to assign the size of the component ?
Like :
```size = VtmnLoaderSize.MEDIUM```

Reply:
Mhhhh this doesn't work like that ;) 
That's an enumeration, not an object